app/components/autocomplete/slash_suggestion/app_command_parser/app_command_parser.ts

```diff
@@ -978,12 +978,15 @@ export class AppCommandParser {
                         const userID = (f.value as AppSelectOption).value;
                         let user: UserModel | UserProfile | undefined = await getUserById(this.database, userID);
                         if (!user) {
-                            const res = await fetchUsersByIds(this.serverUrl, [userID]);
+                            const res = await fetchUsersByIds(this.serverUrl, [userID], true);
```

Comment:
@enahum If the user id provided by the App does not match an existing user, the mobile app crashes here https://github.com/mattermost/mattermost-mobile/blob/068549a2852bff605cad2097697e26b47fe7503e/app/actions/remote/user.ts#L411. What do you suggest as an alternative?

Reply:
Just the last argument leave it as the defsult false so that it get's added to the database and then leave the rest as is

Reply:
@enahum If it's set to false, the mobile app crashes if the user id is not an existing user on the MM server. There has to be some change here to avoid the crash

Reply:
I don't follow, why would it not be an existing user in the server? And in that case, then I think the action should be fixed to handle that

Reply:
@enahum The App can provide any value for any of the fields. If it sets an invalid user id as the initial value of a user autocomplete field, that's when this occurs. I'll look into how to avoid this but idk the specifics of what's happening behind the scenes with `operator.handleUsers`

Reply:
@enahum This is the warning that shows when an invalid user id is provided:

<img width="300" alt="image" src="https://user-images.githubusercontent.com/6913320/203600337-54d26960-d69a-4cdc-9982-aee8a579005e.png">

Reply:
I have this fixed in the latest commit